packages/react-components/react-toast/src/contexts/toastContext.tsx

```diff
@@ -1,18 +1,11 @@
 import * as React from 'react';
-import { ToastIntent } from '../state/types';
 
 export type ToastContextValue = {
-  close: () => void;
-  intent: ToastIntent | undefined;
-  titleId: string;
-  bodyId: string;
+  appearance?: 'inverted';
```

Comment:
why optional? also, there's no default appearance? so if `undefined` is the value for default appearance, perhaps let's used something like:

```suggestion
  appearance: 'inverted' | undefined;
```